sda-commons-server-kafka/src/main/java/org/sdase/commons/server/kafka/health/KafkaHealthCheck.java

```diff
@@ -31,6 +31,7 @@ protected Result check() {
     try (AdminClient adminClient = AdminClient.create(KafkaProperties.forAdminClient(config))) {
       adminClient.listTopics().names().get(2, TimeUnit.SECONDS);
     } catch (Exception e) {
+      logger.warn("Kafka health check failed", e);
```

Comment:
We should not use the logger from `AdminClientConfig` but an own logger for this class.

Reply:
oh yes, you are absolutely right. Thanks for your review.